### ISSUE TITLE
Bold or not bold?

### ISSUE THREAD

**Issue by davidhassell** [cf-convention/cf-conventions#32]
Hi,

There is some confusion as to when monospaced text (`blah`) should be bold (**`blah`**) or not - sometimes it is, sometimes not. Do we need a scheme, like attributes names (and some other things) = bold, attribute values (and some different things) = not bold?

What do you think?

All the best,

David

**Comment by rhattersley** [comment]
Judging by appendix C of the [original HTML](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.html#standard-name-modifiers) and [PDF](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.pdf#page=74) versions and the [new HTML](http://cf-metadata.github.io/#standard-name-modifiers) version, none of them make a visual distinction between normal and bold monospaced text. So from "the outside" it doesn't much matter what we do. :wink: 

It seems silly and/or irresponsible to devote time and effort trying to maintain a distinction which returns no value (and which will be extra hard to maintain since there's no visual feedback!) So that suggests either:

1. Do nothing.
 * Minimal effort. :+1:
2. Remove all the unnecessary bold markup, e.g. ``**`id`**`` becomes `` `id` ``.
 * Minimal "line noise" in source. :+1: 

With my pragmatic hat on I'd suggest sticking with (1) until everything else is fixed.

**Comment by rhattersley** [comment]
Something like:
```
    find | grep adoc | xargs sed -r --in-place 's/\*\*(`[^`]+`)\*\*/\1/g'
```
should take care of (2) once all the other PRs are merged (to avoid unnecessary merge conflicts).

**Comment by dblodgett-usgs** [comment]
Fixed? If 1.7 is out this must be fixed... ?
